index.html

```diff
@@ -19,8 +19,8 @@
       <h1 class="hero-heading">Cat Match</h1>
       <p class="hero-sub-heading">Find your perfect cat breed</p>
       <div class="hero-svg-section">
-        <img src="./assets/images/man_cat_couch.svg" class="hero-svg">
-        <button class="hero-button">Get started</button>
+        <img src="./assets/images/man_cat_couch.svg" class="hero-svg" alt="man and cat on couch">
```

Comment:
hey @Jaytru1997, I'm not sure if that is a man 😅 but we need to be a biiiit more descriptive, maybe something like 'a person sitting on a purple couch with a smiling cat on their lap'? this might be too long of a description but have a look and see what you can come up with